styles.css

```diff
@@ -5,3 +5,27 @@
   margin: 0;
   padding: 0;
 }
+
+:root {
+  --logo-background: #0a0a23;
+}
+
+nav {
+  position: absolute;
+  width: 100%;
+  height: 80px;
+  background: var(--logo-background);
+  top: 0;
+  left: 0;
+  display: grid;
+  place-items: center;
+}
+
+img {
+  width: 336px;
+  height: 40px;
+}
```

Comment:
Let's rename this to something like logo or fcc-logo and make it a little bit smaller. I don't think it needs to be as small as it is on the actual forum page but just a little bit less.